commands/channel_e2e_test.go

```diff
@@ -231,6 +231,44 @@ func (s *MmctlE2ETestSuite) TestCreateChannelCmd() {
 	})
 }
 
+func (s *MmctlE2ETestSuite) TestArchiveChannelsCmdF() {
+	s.SetupTestHelper().InitBasic()
+
+	s.Run("Archive channel", func() {
```

Comment:
We can use to test with local mode as well.
```suggestion
	s.RunForSystemAdminAndLocal("Archive channel", func(c client.Client) {
```

Reply:
hi @isacikgoz , this fails the entire test case as the `archiveChannelsCmdF` function returns an error with a local client. If that is the intended functionality, suggest that we should split the tests so that we can explicitly pass in different clients into different test cases, as per what was implemented in the e2e tests in `unarchiveChannelsCmdF`

Reply:
Speaking of which, the error returned from the local client is "The channel has been archived or deleted", while theoretically speaking, though I may be wrong, it should return a permissions related error message.

This could be an issue that can be addressed in a separate PR.